.github/workflows/arm64-build.yml

```diff
@@ -0,0 +1,55 @@
+name: ARM64 Build
+
+on:
+  pull_request:
+
+jobs:
+  build-arm64:
```

Comment:
Unless your intention is to make this optional, it may be better to extend the existing `ci.yml` and do a matrix build if possible